src/databricks/labs/ucx/source_code/linters/python_ast.py

```diff
@@ -192,191 +198,29 @@ def _get_attribute_value(cls, node: Attribute):
             logger.debug(f"Missing handler for {name}")
         return None
 
-    def infer_values(self, state: CurrentSessionState | None = None) -> Iterable[InferredValue]:
```

Comment:
this change is a bit more difficult to review, as methods are moved, the reasoning is not clear

Reply:
methods are moved to avoid work around cyclic imports